configs/projects/agent-runtime-main.rb

```diff
@@ -79,6 +79,6 @@
   proj.component 'rubygem-erubi'
   proj.component 'rubygem-prime'
 
-  proj.component 'boost' if ENV['NO_PXP_AGENT'].to_s.empty?
-  proj.component 'yaml-cpp' if ENV['NO_PXP_AGENT'].to_s.empty?
+  proj.component 'boost' if ENV['NO_PXP_AGENT'].to_s.empty? && platform.name != 'solaris-113-sparc'
+  proj.component 'yaml-cpp' if ENV['NO_PXP_AGENT'].to_s.empty? && platform.name != 'solaris-113-sparc'
```

Comment:
Same question for these components, are planning on coming back and adding these, or are they not needed?